app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,39 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name", class: "input" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :bio %><br />
```

Comment:
![Selection_069](https://github.com/Chimoneg27/Rails-Blog-App/assets/107632780/f9dd6edf-7135-4de2-8c81-0bd25f2fecbc)

- [ ] As stated in the project requirement and pointed out by the previous reviewer, you need to style your devise screens `Login and Registration` page to look like the wireframe. :+1: You have styled the login screen but the `signup page` has not been styled. I'll suggest you style the signup page to look like the wireframe. :wave: :small_airplane: 

- - Your current Design:
![Screenshot from 2023-12-15 01-32-44](https://github.com/Chimoneg27/Rails-Blog-App/assets/107632780/4ed8b163-9a42-4159-9c9b-b8aff926aab6)